src/Native/Unix/configure.cmake

```diff
@@ -362,6 +362,34 @@ check_prototype_definition(
 
 check_cxx_source_compiles(
     "
+    #include <stdlib.h>
+
+    int main()
+    {
+        char* path = strdup(\"abc\");
+        int ret = mkstemps(path, 3);
```

Comment:
Why is the strdup necessary rather than just passing a string to mkstemps?  We're not running it, just compiling it.